process/smartContract/process.go

```diff
@@ -936,13 +935,17 @@ func (sc *scProcessor) treatExecutionAfterBuiltInFunc(
 		return true, userErrorVmOutput, newVMInput, sc.ProcessIfError(acntSnd, vmInput.CurrentTxHash, tx, err.Error(), []byte(""), snapshot, vmInput.GasLocked)
 	}
 
-	err = sc.checkUpgradePermission(acntDst, newVMInput)
+	newDestSC, err := sc.getAccountFromAddress(vmInput.RecipientAddr)
```

Comment:
Is this backwards compatible?

Reply:
yes. it is. there is no execution after built in only after esdt is enabled.